doctr/datasets/synthtext.py

```diff
@@ -72,9 +72,12 @@ def __init__(
                 raise FileNotFoundError(f"unable to locate {os.path.join(tmp_root, img_path[0])}")
 
             labels = [elt for word in txt.tolist() for elt in word.split()]
-            word_boxes = word_boxes.transpose(2, 1, 0) if word_boxes.ndim == 3 else np.expand_dims(word_boxes, axis=0)
+            # (x, y) coordinates of top left, top right, bottom right, bottom left corners
+            word_boxes = word_boxes.transpose(2, 1, 0) if word_boxes.ndim == 3 else np.expand_dims(
+                word_boxes.transpose(1, 0), axis=0)
```

Comment:
This isn't related to the PR description: are you adamant that this is correct?

Reply:
@fg-mindee 
Yes i know and yes i have tested it on the original dataset
Before:
(2, 4) -> (1, 2, 4)
After:
(4, 2) -> (1, 4, 2)